pulsar-client-cpp/lib/Commands.cc

```diff
@@ -308,6 +309,14 @@ SharedBuffer Commands::newSubscribe(const std::string& topic, const std::string&
         subscribe->mutable_metadata()->AddAllocated(keyValue);
     }
 
+    for (std::map<std::string, std::string>::const_iterator it = subscriptionProperties.begin();
+         it != subscriptionProperties.end(); it++) {
```

Comment:
I think we can use the C++11 style for new code. Though some legacy code still uses older style. We can do some refactoring to these legacy code later.

Reply:
thanks for your suggestion, it has been revised.